source/_integrations/squeezebox.markdown

```diff
@@ -167,34 +176,68 @@ data:
     announce_timeout: 60
 ```
 
-## Entities
+## Supported functionality
+
+### Entities
+
+The integration provides the following entities.
+
+#### Binary sensors
+
+- **Needs restart**
+  - **Description**: Server Service needs to be restarted (typically, this is needed to apply updates).
+
+- **Library rescan**
+  - **Description**: The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
+
+#### Buttons
+
+- **Preset 1 ... Preset 6**
+  - **Description**: Play media stored in Preset 1 to Preset 6 on Squeezebox.
+
+- **Brightness Up, Brightness Down**
+  - **Description**: Adjust the brightness on Logitech Squeezebox players
+  - **Available On**: Logitech hardware players with built-in screen, such as Radio and Boom.
+  
+- **Bass Up, Bass Down**
+  - **Description**: Adjust the bass on Logitech Squeezebox players, such as Radio and Boom.
+  - **Available On**: Logitech hardware players such as Radio, Duet and Boom.
```

Comment:
```suggestion
  - **Available on**: Logitech hardware players such as Radio, Duet and Boom.
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization), also for UI strings and headings.